api/schemas/report.json

```diff
@@ -114,6 +114,11 @@
             ]
         },
 
+        "master": {
+            "description": "The name of the master that was used to compile the catalog. If failover occurred, this will hold the first master successfully contacted. If this run had no master (e.g. a `puppet apply` run), this field will be blank",
```

Comment:
So when using the `server` setting and specifying a bogus master and `report_server` to localhost, I get a report with the bogus master in it. If this is not the functionality we want, we'll have to move where I'm doing the reporting. 

When using `server_list`, no report is generated. Could this be a bug with the way we're handling server_list? I know we had questions about how it would affect other servers the user had specified.